Classes/Issues/EditComment/EditCommentViewController.swift

```diff
@@ -164,4 +165,9 @@ class EditCommentViewController: UIViewController {
         ToastManager.showGenericError()
     }
 
+    func textViewDidChange(_ textView: UITextView) {
+        guard textView === self.textView else { return }
```

Comment:
Just as a sanity check in case we add another textview, but can remove if redundant.

Reply:
Ya bit redundant atm